WordPress/Classes/ViewRelated/Comments/CommentsViewController+NetworkAware.swift

```diff
@@ -0,0 +1,26 @@
+import UIKit
+
+
+// CommentsViewController is an Objective-C class, so in order for interop to work, it looks like we need to override these methods.
```

Comment:
This is not clear to me, are you saying that you need to override the methods to add the `objc` part so they can be used in `CommentsViewController`?

Reply:
I am not sure how to word that better, so I removed it for now.